pkg/controller/constrainttemplate/stats_reporter.go

```diff
@@ -15,19 +15,21 @@ const (
 	ctCount        = "total_constraint_templates"
```

Comment:
Drop total here too: "constraint_templates"

Reply:
What about consistency with the metrics `total_constraints` and `total_violations`?

Reply:
Consistency is important. If possible, you should consistently follow the best practices / standards:
https://prometheus.io/docs/practices/naming/

If you can't go back and update the names of those metrics, then next best alternative would be consisten in not following them (aka leave as-is).

Reply:
Removing `total` from all metric names.

@sozercan LMK when you have a chance to review.

Reply:
sounds good to me